monad-logger/Control/Monad/Logger.hs

```diff
@@ -428,6 +432,88 @@ instance MonadUnliftIO m => MonadUnliftIO (NoLoggingT m) where
                 withUnliftIO $ \u ->
                 return (UnliftIO (unliftIO u . runNoLoggingT))
 
+-- | @since 0.3.28
+newtype WriterLoggingT m a = WriterLoggingT { unWriterLoggingT :: m (a, DList LogStr) }
```

Comment:
I think it would make more sense to keep the `LogLevel` and other values that are included in the logging implementation, instead of simply discarding them. A user may of course choose to throw them away.